From 5d55c17a9f0b1237c78c3fd6c7f8f893cdbfb398 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Fri, 13 Nov 2015 21:32:34 -0800
Subject: [PATCH 093/194] SnapdragonCamera: Unlock AE/AWB after taking a photo
 with ZSL

The exposure and white balance get stuck after taking photos with ZSL.

Always unlock the exposure and white balance after taking photos with
ZSL.

Change-Id: Ia34a54522048fdd619f44ac1f6d2924ce2c3fc6a
---
 src/com/android/camera/PhotoModule.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/camera/PhotoModule.java b/src/com/android/camera/PhotoModule.java
index 01c4aae34..cc3c1e25e 100644
--- a/src/com/android/camera/PhotoModule.java
+++ b/src/com/android/camera/PhotoModule.java
@@ -2853,6 +2853,7 @@ public class PhotoModule
         if (null != mCameraDevice ) {
             mCameraDevice.cancelAutoFocus();
             setCameraState(IDLE);
+            mFocusManager.setAeAwbLock(false);
             setCameraParameters(UPDATE_PARAM_PREFERENCE);
         }
     }
-- 
2.17.1

